### PR TITLE
Push the latest code style updates for Ink Recognizer C# UWP Sample

### PR DESCRIPTION
**Pull request by gowthu2712**
## Purpose

* Push the latest code style updates for C# UWP sample

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[X] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
Build the NoteTaker.sln
```

* Test the code

```
1) Add subscription key to cs\NoteTaker.xaml.cs
2) Ink some text / draw a shape on the Inking area
```

## What to Check
Verify that the following are valid
```
Make sure the recognized text/shape is displayed at the bottom of the app
```

### CODE REVIEW

File: dotnet/InkRecognition/uwp-app/cs/Services/Ink/InkRecognizer.cs [hunk 1]
```diff
@@ -1,81 +1,81 @@
...
+namespace Contoso.NoteTaker.Services.Ink
+{
+    public class InkRecognizer
+    {
+        List<InkRecognizerStroke> strokes;
+        InkRecognitionRoot root = null;
+
+        HttpManager httpManager;
+
+        // Default DPI setting
```
Comment (olduroja) [inline dotnet/InkRecognition/uwp-app/cs/Services/Ink/InkRecognizer.cs]: perhaps you should given a reason why.

File: dotnet/InkRecognition/uwp-app/README.md [hunk 2]
```diff
@@ -1,54 +1,60 @@
...
-Select Build > Deploy Solution.
-
-### Debugging and running the sample
-To debug the sample and then run it, press F5 or select Debug > Start Debugging. To run the sample without debugging, press Ctrl+F5 or select Debug > Start Without Debugging.
+---
+topic: sample
+languages:
+  - C#
+products:
+  - WPF
```
Comment (olduroja) [inline dotnet/InkRecognition/uwp-app/README.md]: This is the UWP app right? WPF shouldn't be here.

Comment (gowthu2712) [reply dotnet/InkRecognition/uwp-app/README.md]: Changed in the latest commit

File: dotnet/InkRecognition/uwp-app/cs/Services/Ink/InkRecognizer.cs (hunk 1 above)
Comment (gowthu2712) [reply dotnet/InkRecognition/uwp-app/cs/Services/Ink/InkRecognizer.cs]: There is comment explaining when to use default setting in L50-51 of this file

Comment (xinyzhou) [reply dotnet/InkRecognition/uwp-app/cs/Services/Ink/InkRecognizer.cs]: [Wiki for DPI](https://en.wikipedia.org/wiki/Dots_per_inch) says Microsoft Windows use 96 as default DPI, [This blog](https://blogs.msdn.microsoft.com/fontblog/2005/11/08/where-does-96-dpi-come-from-in-windows/) also gives a explaination. But anyway, I think it is good enough to say it's windows default settings.

Comment (olduroja) [reply dotnet/InkRecognition/uwp-app/cs/Services/Ink/InkRecognizer.cs]: @xinyzhou, the value of the default isn't the issue. The fact that the current dpi may be something else is. Someone looking at the code for the first time may wonder why you're hard coding the default. @gowthu2712 , please add a reference to the other comment.

**Pull request merged**